### PR TITLE
feat(odk): custom endpoint for ODK Collect urls

### PR DESCRIPTION
**Pull request by obdulia-losantos**
This could be useful with the kong option, instead of indicating all the "non-protected" ODK Collect urls, include the `ODK_COLLECT_ENDPOINT="odk-collect/"` environment variable and just indicate this endpoint in the non-protected list. Take in mind that if ODK Collect renames the endpoints (unlikely) or adds a new one (unexpected) we don't need to redefine the kong configuration.

### CODE REVIEW

**Review by shawnsarwar — DISMISSED** [review]
Nice idea. I like unifying all the unprotected (by OIDC) endpoints unto a single path away from everything else, certainly cuts down on config.

File: README.md [hunk 1]
```diff
@@ -289,6 +289,9 @@ The default values for the export feature:
 - `AETHER_KERNEL_TOKEN_TEST`: `kernel_any_user_auth_token` Token to connect to testing kernel server.
 - `AETHER_KERNEL_URL`: `http://aether.local/kernel/` Aether Kernel Server url.
 - `AETHER_KERNEL_URL_TEST`: `http://kernel-test:9100` Aether Kernel Testing Server url.
+- `ODK_COLLECT_ENDPOINT`: the endpoint for all ODK collect urls.
+  If it's `collect/` the submission url would be `http://my-server/collect/sumbission`
```
Comment (shawnsarwar) [inline README.md]: `sumbission`

Comment (obdulia-losantos) [reply README.md]: :see_no_evil:

**Comment by obdulia-losantos** [comment]
@shawnsarwar could you review it again? :grimacing:

**Review by shawnsarwar — APPROVED** [review]

**Pull request merged**